src/modules/transaction/hooks/useTransactionFee/utils.js

```diff
@@ -16,19 +16,26 @@ import { fromTransactionJSON } from '../../utils/encoding';
  * @returns {bigint} the transaction fee in Beddows
  */
 
-export const computeTransactionMinFee = (transactionJSON, paramsSchema, numberOfSignatures, extraCommandFee) => {
+export const computeTransactionMinFee = (
+  transactionJSON,
+  paramsSchema,
+  numberOfSignatures,
+  extraCommandFee
+) => {
   const options = {
     numberOfSignatures,
     numberOfEmptySignatures: 0,
     additionalFee: BigInt(extraCommandFee),
   };
-  const minFee = transactions.computeMinFee(
-    fromTransactionJSON(transactionJSON, paramsSchema),
-    paramsSchema,
-    options
-  );
 
-  return minFee;
+  let convertedTx = {};
+  try {
+    convertedTx = fromTransactionJSON(transactionJSON, paramsSchema);
+  } catch (exp) {
+    convertedTx = {};
```

Comment:
Isnt it already an empty object?
Maybe initialise it with
`let convertedTx;` 

Or not mutating it at all
```
  try {
    const convertedTx = fromTransactionJSON(transactionJSON, paramsSchema);
    
    return transactions.computeMinFee(convertedTx, paramsSchema, options);
  } catch (exp) {
    return transactions.computeMinFee({}, paramsSchema, options);
  }
```